### PR TITLE
Update test-mbamwebapplication.md

### PR DESCRIPTION
**Pull request by msbemba**
added description to -DisableNoticePage

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/mdop/mbam/test-mbamwebapplication.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ Accept wildcard characters: False
 ```
 
 ### -DisableNoticePage
-{{Fill DisableNoticePage Description}}
+Indicates the Self-Service Portal notice text on or off. By default, the notice text is turned on.
```
Comment (JohanFreelancer9) [inline docset/mdop/mbam/test-mbamwebapplication.md]:
```suggestion
Indicates whether the Self-Service Portal notice text is turned on or off. By default, the notice text is turned on.
```

**Comment by get-itips** [comment]
I am ok once the copy-review is applied.

**Pull request merged**